scripts/build/build_examples.py

```diff
@@ -192,14 +153,19 @@ def cmd_generate(context):
     'targets',
     help=('List the targets that would be generated/built given '
           'the input arguments'))
+@click.option(
+    '--expand',
```

Comment:
I see a lot of `'nodeps' does not support '` when I run this, dwarfing the actual visible output. Perhaps we should suppress that error?

Reply:
I will do this as a followup. I believe for --expand it makes sense, but for others I would like to keep it since it is not obvious why in some cases a target does not exist.